README.md

```diff
@@ -47,6 +47,8 @@ package main
 
```

Comment:
While you're here: might as well append `go` to the three backticks on line 45 to get the correct syntax highlighting.